Formula/re-flex.rb

```diff
@@ -0,0 +1,30 @@
+class ReFlex < Formula
+  desc "The regex-centric, fast and flexible scanner generator for C++"
+  homepage "https://www.genivia.com/doc/reflex/html"
+  url "https://github.com/Genivia/RE-flex/archive/reflex-1.0.12.tar.gz"
+  sha256 "75a1761d9a6fd3fee6454e3b6ec954efbad76ece9b68633a07ada977af4cd300"
+
+  depends_on "boost"
+
+  def install
+    system "./configure",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "-j", "install"
```

Comment:
This is wrong, `-j` option should not be used, it is automatically set by `brew`.

Reply:
Hey guys, any progress on reviewing the updates we made 4 days ago?